githubapi/example/index.js

```diff
@@ -0,0 +1,35 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const { Octokit } = require("@octokit/rest");
+const { request } = require("@octokit/request");
+
+
+const octokit = new Octokit({
+    auth: "access_token",
+    userAgent: 'actions test',
+    timeZone: 'Europe/Amsterdam',
+    baseUrl: 'https://api.github.com',
+
+    log: {
+        debug: () => {},
+        info: () => {},
+        warn: console.warn,
+        error: console.error
+      },
+      request: {
+        agent: undefined,
+        fetch: undefined,
+        timeout: 0
+      }
+    })
+
+async function TestRequest() {
+  const TestData = octokit.repos.listContributors({

```

Comment:
This returns a `Promise`. You either have to return the promise or `await` it.

Reply:
Here is an example of how you should await a promise on the top level of execution:
https://github.com/ory/closed-reference-notifier/blob/master/src/runInAction.ts#L22-L36